Conference Website development

ProgramKingAbel opened the pull request:
- Home and About page developed using mobile first approach.
- desktop versions implemented using media queries.
- dynamically generating speakers section and see-more button in javascript
- Functional Mobile menu.
- deployment done from branch on github pages

ProgramKingAbel committed:
Completed homepage mobile version

ProgramKingAbel committed:
About page mobile section

ProgramKingAbel committed:
Added Mobile Menu

ProgramKingAbel committed:
Implemented desktop version for index and about markups

ProgramKingAbel committed:
Generated Speaker section dynamically in js

ProgramKingAbel committed:
Create CNAME

victorokoroji commented on README.md:
```diff
@@ -24,7 +24,7 @@
 # 📖 [Abel-Morara-Portfolio] <a name="about-project"></a>
```
- [ ] Kindly ensure that you give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/) In the Acknowledgements" section of your readme 😄 .

victorokoroji commented on README.md:
```diff
@@ -52,24 +52,22 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Go to Live site](https://programkingabel.github.io/)
+- [Go to Live site](http://conference.org/)
```
- [ ] Kindly ensure you use GitHub Pages to deploy the web pages and add the working link to your readme 👍🏽

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+                <a href="#">My Page</a>
+                <a href="#">Logout</a>
+    
+                </div>
```
- [ ]I think it will be better if you wrap the elements of your top nav In a list (ul tag) just as you have done for the bottom nav. This is a semantic way of organizing the top navigation elements since they are represented as a list of top av elements 👍🏽

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+    <section class="hero">
+        <div class="hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT <br />2015</h1>
```
- [ ] I think you should consider increasing the font size, and font weight of your hero title so that it matches the design guideline you are given. I think you should consider using `cocogoose` font instead of `lato` which in my opinion is the right font for the hero title. 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235219483-5f037eb0-b246-4b4f-a6d1-a47f19afcc1e.png)

design guideline 👇🏽
![image](https://user-images.githubusercontent.com/63473584/235219642-94052d5d-79fb-4f27-8d96-61756fc8665c.png)

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+        <div class="hero-wrapper">
+        <p class="hello">"Hello Sharing world</p>
+        <h1>CREATIVE COMMONS <br /> GLOBAL SUMMIT <br />2015</h1>
+        <p class="description">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Korea.</p>
```
- [ ] I think you should consider reducing the width of your introductory paragraph to match the design given to you 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235220499-f51154fe-d2c5-448e-910d-5b146902fcd2.png)

design guideline 👇🏽
![image](https://user-images.githubusercontent.com/63473584/235220372-9c25986b-0188-459b-8473-42461ad4ca63.png)

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+        </div>
+    </section>
+
+    <section class="speakers">
```
- [ ] Good job on rendering your featured speakers dynamically 👍🏽 . I just noticed that on the page load, only two featured speakers is displayed. Kindly modify your functionality so that the six featured speakers and their descriptions appear on the desktop view when the page is visited so that it matches the design you are given.  

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235222396-9285b51f-c230-401e-ba56-561db07e3b55.png)

design guildeline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235222669-a25c8eb4-9fcb-46f4-8f11-053f1986ec2d.png)

ProgramKingAbel committed:
Implemented show more functionality on button

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+            <button href="#" onclick="loadData()" class="see-more">MORE<span></span><i class="uil uil-angle-down"></i></button>
+        </div>
+
+        <section class="partner">
+            <h2>Partner</h2>
+            <hr />
+            <div class="partner-logos">
+                <img src="./images/airbnb.svg" alt="">
+                <img src="./images/daumkakaologo.svg" alt="">
+                <img src="./images/googlekorea.svg" alt="">
+                <img src="./images/mozilla.svg" alt="">
+                <img src="./images/naver.svg" alt="">
+            </div>
+        </section>
```
- [ ] KIndly add appropriate spacing between the elements of your partner's section. They look too close together in your design which is not a good user experience. Kindly modify it

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235223222-50eb637c-6853-450b-a39a-339d64cdd872.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235223399-e5a9cd52-bc54-45cf-a4b1-92adc0b7ffbc.png)

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+            </div>
+        </section>
     
+        <footer>
+            <img src="./images/cck-logo1.svg" alt="">
+            <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+        </footer>
```
- [ ] KIndly add appropriate spacing between the elements of your footer section. They look too close together in your design which is not a good user experience. Kindly modify it

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235223894-49eba6f7-2659-4db2-b9bd-8c01dc7c72f0.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235223945-e1d476ba-9103-4f2e-882e-35322d082c38.png)

victorokoroji commented on index.html:
```diff
@@ -7,12 +7,143 @@
+    <section class="program">
+        <div class="program-wrapper">
+
+            <h2 class="program-title">Main Program</h2>
```
- [ ] KIndly check out the way the underline for each section title are designed and the color assigned to them. Yours look different

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235224678-5b263a94-7bcd-470a-9a4b-4e6318c96db1.png)

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235224827-6942e0b6-899a-4ac1-9140-a30741ccc1f2.png)

ProgramKingAbel committed:
Changed a tag to button in speaker section

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,130 @@
+        </div>
+    </section>
+
+    <footer>
+        <img src="./images/cck-logo1.svg" alt="">
+        <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+    </footer>
```
- [ ] Please note that the footer for the About page has a dark background with white text and more so consider adding appropriate padding or spacing around  the footer elements to match the design guideline

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235225627-afc51436-9304-4c95-b4fd-b9122e69c84b.png)

design guildeline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235225796-a9116322-3309-4079-bfa6-cd62f60c2d94.png)

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,130 @@
+
+    </section>
+
+    <section class="partner">
+        <h2>Partner</h2>
+        <hr />
+        <div class="partner-logos">
+            <img src="./images/airbnb.svg" alt="">
+            <img src="./images/daumkakaologo.svg" alt="">
+            <img src="./images/googlekorea.svg" alt="">
+            <img src="./images/mozilla.svg" alt="">
+            <img src="./images/naver.svg" alt="">
+        </div>
+    </section>
```
- [ ] The About page does not have a partners section so I will recommend you remove it so as to stick to the design given to you 👍🏽

victorokoroji commented on about.html:
```diff
@@ -0,0 +1,130 @@
+            <p>Take a look at the past two
+             CC Global Summits which took place in Buenos Aires and in Warsaw</p>
+             <div class="all-testimonials">
+             <div class="testimonial">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">Global Summit 2011 in Warsaw</p>
+            </div>
+
+             <div class="testimonial">
+             <h3 class="year">2011</h3>
+             <p class="about-summit">Global Summit 2011 in Warsaw</p>
+            </div>
```
- [ ] Kindly reduce the width of your testimonial cards ad centralize them so as to match the design guideline 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235227086-f6cd1a7a-8aab-4d23-9fe6-3243b9f97865.png)

design guideline 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/235227018-d5a78704-e3be-4c4b-b6b6-86ea1ac190db.png)

victorokoroji reviewed (changes requested):
Hi @ProgramKingAbel, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Good video presentation✅ 
- Good use of sematic tags ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] Please take special note that the project requirement specifies that you personalize the content of your webpage so that the webpage will be unique to you, but rather you repeated the same texts on the design guideline. Kindly research your own topic of interest and personalize the content of your webpage using the same design 😄 

_Check the comments under the review._
<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ProgramKingAbel committed:
Updated README.md

ProgramKingAbel committed:
Merge branch 'site-development' of github.com:ProgramKingAbel/Conference-site- into site-development

ProgramKingAbel committed:
Delete CNAME

ProgramKingAbel commented:
I have made changes are requested by reviewer Vickymaz. in the following ways:
- added a functional deployed project link with GH Pages.
- adjusted content sizing and spacing.
- added functionality on dynamically generating speakers depending on screen size
- Personalized content
- Improved README.MD on acknowledgments.
- adjusted footer features as rendered on different screens. 

Thank you for your guidance and your time

ProgramKingAbel committed:
Adjust content with js dynamically with screen-size on load

ProgramKingAbel committed:
Personalized content and finalized reviewer changes

ProgramKingAbel committed:
Personalized content and finalized reviewer changes

KanzaTahreem commented on README.md:
```diff
@@ -211,6 +208,8 @@ If you like this project follow me on Github and give it a star to motivate buil
 
 I would like to thank the Microverse Community, My stand up team and coding partners who we work tirelessly to make it possible.
 
+Lots of love and appreciation to [Cindy Shin] (https://www.behance.net/adagio07) the [author of the original design] (https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), in Creative Commons platform.
```
- [OPTIONAL] Well done for giving the appropriate credits to Cindy Shin. What do you think of removing the space between these brackets 👉 `[ ]( )` ✨

ProgramKingAbel committed:
Merge branch 'site-development' of github.com:ProgramKingAbel/Conference-site- into site-development "Solving merge errors"
git commit -m "Solving merge error"

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+    position: relative;
+  }
+
+  nav .top {
+    display: flex;
+    width: 100%;
+    align-items: center;
+    justify-content: flex-end;
+    margin: 0 auto;
+    background: #3e3e3e;
+    padding: 0.2rem 1rem;
+    gap: 1rem;
+  }
+
```
- [OPTIONAL] Please consider aligning the top nav with the bottom navbar according to the design guidelines 🔨✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235278840-11f9a66f-54a7-45ca-994c-d633d203326c.png) | ![image](https://user-images.githubusercontent.com/106119105/235278900-ef8a586e-6a2d-4842-9b83-411e17c28e0b.png)

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+    color: #868686;
+  }
+
+  .hero-wrapper h1 {
+    font-size: 2.5rem;
+    font-weight: 800;
+    line-height: 3rem;
+    background-clip: initial;
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    background-image:
+      -webkit-linear-gradient(
+        45deg,
+        #f40001,
+        #ec5242,
+        rgb(255, 64, 0),
+        rgb(255, 123, 0),
+        #a00000,
+        #f65112,
+        #ec5242
+      );
+  }
```
- [ ] Good job increasing the `font-weight`. However, as suggested by the reviewer [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180677569), please make sure to increase the `font-size` also 💪

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  padding: 5rem 1rem 5rem 0;
+}
+
+.hello {
+  font-family: 'Lato', sans-serif;
+  color: #ff5438;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1rem;
+}
```
- [OPTIONAL] I think it would be better to implement the following for **_"Hello young entrepreneurs"_** for desktop version 👇 

```
  font-size: 1.7rem;
  letter-spacing: 0.5px;
  padding-bottom: 0.5rem;
```

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+      );
+  }
+
+  .description {
+    width: 68%;
+    font-weight: 400;
+    font-size: 1.3rem;
+    line-height: 1.6rem;
+  }
+
```
- [OPTIONAL] What do you think of reducing the `font-size` of the description to `1rem` for desktop version💭😅

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  text-align: left;
+  margin-bottom: 2rem;
+}
+
+.date {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 700;
+  font-size: 1.1rem;
+  color: #595959;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
```
- [OPTIONAL] I think it would be better to reduce the `font-size` to `2rem` from `2.7rem` for desktop version. Also, please consider using the `font-family: Lato` as recommend in the design guidelines with the following 👇🔨
 ```
  letter-spacing: 0.5px;
  font-weight: 800;
```

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  padding-bottom: 0.2rem;
+}
+
+.location {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 600;
+  font-size: 0.7rem;
+  color: #868686;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
```
- [OPTIONAL] Similarly here, I think it would be better to use the `font-family` given in the design guidelines 💪 Also, what do you think of implementing the following for desktop version? 🤔

```
  font-size: 1.1rem;
  letter-spacing: 0.5px;
```

KanzaTahreem commented on index.html:
```diff
@@ -7,12 +7,147 @@
+            </div>
+        </section>
     
+        <footer>
+            <img src="./images/cck-logo1.svg" alt="">
+            <p>2015 Creative Commons Korea. Some Rights Reserved.</p>
+        </footer>
```
- [ ] As suggested by the previous reviewer, please make sure to personalize the content of your webpage ✨ You have done a great job in personalizing other sections, please personalize this as well 😄

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  color: #fff;
+  padding: 0.5rem;
+}
+
+.program-wrapper hr {
+  width: 15%;
+  border: 2px solid #a04038;
+  margin-bottom: 2rem;
+}
```
- [ ] As suggested by the reviewer [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180701352), please fix the line and it's color after **_Main Program_**. The color given in the design guidelines is `#ec5242`. Please fix this 🛠🤓


Your design | Design guidelines
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/235274254-2a051bf3-aa57-4e33-8a0f-769f6f538530.png) | ![image](https://user-images.githubusercontent.com/106119105/235274285-4f88cf3c-0577-41a7-9683-2b0a89191ab1.png)

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  text-align: center;
+}
+
+.speaker-wrapper hr {
+  width: 45px;
+  border: 1.5px solid #a04038;
+  margin-bottom: 1rem;
+}
```
- [ ] Please fix the color of the line here as well as suggested [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180701352) 📌

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info .s-title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 1rem;
+  color: #f78c89;
+  margin-bottom: 0.2rem;
+}
+
```
- [OPTIONAL] Please consider adding `font-style: italic;` here 🤓🔧I'm sure this will definitely enhance your design ✨

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+    margin-top: 1rem;
+    text-decoration: underline;
+  }
+
+  .all-speakers {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 0.5rem;
+  }
+
```
- [OPTIONAL] I think it would be better to add some padding to the top. What do you think? ✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235274577-51a722a5-705f-4a04-9144-c76650f5631e.png) | ![image](https://user-images.githubusercontent.com/106119105/235274602-1bc1a9e4-1f60-46ea-851f-2124428ac6cd.png)

KanzaTahreem commented on index.css:
```diff
@@ -0,0 +1,540 @@
+    gap: 1rem;
+    flex-wrap: wrap;
+  }
+
+  footer {
+    width: 100%;
+    margin: 0 auto;
+    padding: 4rem 0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    gap: 10%;
+  }
```
- [ ] As suggested by the reviewer [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180706453), please make sure to add appropriate padding/spacing around the footer elements to match the design guideline. Right now, the `footer` has way a lot of padding 😓Please reduce this ⚒

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235275841-ff8398d8-6417-4d0d-acce-8607cbca9877.png) | ![image](https://user-images.githubusercontent.com/106119105/235275750-15d9280b-8651-4a3b-ac25-a52da71697b5.png)

KanzaTahreem commented on about.css:
```diff
@@ -0,0 +1,435 @@
+    aspect-ratio: 2/1;
+  }
+
+  .all-testimonials {
+    width: 80%;
+    margin: 0 auto;
+    flex-direction: row;
+  }
```
- [ ] You have done a good job by reducing the width of these cards 👍👍 However, kindly reduce the width a bit more to match the design guidelines as suggested [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180711126)
- [OPTIONAL] What do you think of adding some space/padding/margin to the top and bottom of these cards as mentioned in the following images? 💭 I think it will enhance your design ✨

Your design | Design guidelines
---| ---|
![image](https://user-images.githubusercontent.com/106119105/235278059-b5cbeb63-5a25-434a-9896-f2cc09b3a660.png) | ![image](https://user-images.githubusercontent.com/106119105/235278033-38dc6975-be1f-41b4-9e5e-cb00249dba1f.png)

KanzaTahreem commented on index.css:
- [OPTIONAL] Good job with CSS 🥇 I noticed that `font-family: 'Lato', sans-serif;` is repeating a lot. Please make sure to follow the HTML and CSS best practices and avoid repeating it over and over again 😃 Instead, you can assign it under `body` selector. [Check out this link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more information ℹ

KanzaTahreem reviewed (changes requested):
# Changes Required 🔁

Hi @ProgramKingAbel 👋👋

Good job so far implementing the changes suggested by the reviewer! I know you worked hard to implement this project, I really appreciate the effort you put in! 

**_However, there are few things that still need to be fixed. But you are almost there!_** 💪 🏁

![so_close](https://user-images.githubusercontent.com/99715640/211022680-440a1bb1-4281-4acb-8dbc-3442fe509063.gif)

## Highlights 🎯

There are some excellent points that you have made perfectly. Keep up the great work! 👏
- ✅ Good job with the design
- ✅ Great job with CSS
- ✅ Descriptive PR and README file
- ✅ Correct Linters

However, there is always room for improvement.

## Required Changes 🛑

> _Please keep in mind that your long hours of work is appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. 😄_

_Please check the comments under the review._

## Optional suggestions ❕
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

N/A

Cheers and Happy coding!👏👨‍💻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Ibizugbe commented on about.html:
```diff
@@ -0,0 +1,130 @@
+        <h2>Partner</h2>
+        <hr />
+        <div class="partner-logos">
+            <img src="./images/airbnb.svg" alt="">
+            <img src="./images/daumkakaologo.svg" alt="">
+            <img src="./images/googlekorea.svg" alt="">
+            <img src="./images/mozilla.svg" alt="">
+            <img src="./images/naver.svg" alt="">
```
[OPTIONAL] To make your page more semantic, I suggest you add a descriptive value to your `alt` attribute👍. This gives screen readers an idea of what the image is about when presenting it to visually challenged people and also can serve as a means to understand what the image is about when the link to the image is broken or there's a bad network👍

Ibizugbe reviewed (approved):
Hi @ProgramKingAbel 👋👋👋, 

![StandingOvationGIF (4)](https://user-images.githubusercontent.com/64643995/235300302-2f95f9b1-b744-46ea-9c31-8c74c11d5fe5.gif)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] Linters are passing👍.
- [x] changes regarding stylings requested by the previous reviewer have been implemented👍.
- [x] Github flow have been used👍.

## Optional suggestions
_ See comments below😊_
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ProgramKingAbel committed:
Made requested changes by code reviewer

ProgramKingAbel committed:
Solved Stylelint errors

ProgramKingAbel committed:
Adjusted lineheight for hero header
